stable/aws-sigv4-proxy-admission-controller/templates/webhook.yaml

```diff
@@ -1,6 +1,10 @@
 {{ $tls := fromYaml ( include "aws-sigv4-proxy-admission-controller.gen-certs" . ) }}
 ---
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1" }}
```

Comment:
Why do we need this conditional checks ?

Reply:
I assume there would be in a few years when it is no longer used.